tests/test_objects.py

```diff
@@ -258,7 +265,9 @@ def test_GridFsObjects(self):
         conn = yield txmongo.MongoConnection(mongo_host, mongo_port)
         db = conn.test
         gfs = GridFS(db)  # Default collection
-        _ = yield gfs.get("test_3")
+        # Missing file raises error
+        with self.assertRaises(NoFile):
```

Comment:
Unfortunately, `assertRaises()` can be used as context manager only with Twisted ≥ 15 :(
Please rewrite it as `yield self.assertFailure(gfs.get("test_3"), NoFile)`